Scheduler/observing_blocks_maintel/BLOCK-365.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Imaging",
+    "program": "BLOCK-365",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "maintel/track_target_and_take_image_lsstcam.py",
+            "standard": true,
+            "parameters": {
+                "ra": "$ra",
+                "dec": "$dec",
+                "rot_sky": "$rot_sky",
+                "band_filter": "$band_filter",
+                "name": "$name",
+                "estimated_slew_time": "$estimated_slew_time",
+                "exp_times": "$exp_times",
+                "num_exp": "$num_exp",
+                "obs_time": "$obs_time",
+                "program": "$program",
+                "reason": "science",
```

Comment:
It is likely worth getting the update in ts_scheduler so that "reason" can be passed along. That way you could identify DD visits from the Wide visits.

Reply:
Agreed, but maybe for now we can just merge this as-is and update it when the "reason" keyword is available from the scheduler?

Reply:
Agreed, we just need to make sure it is available from the Scheduler (which I think it will be in the next release).